### PR TITLE
pgwar entity class

### PR DESCRIPTION
**Pull request by joschne**
This PR adds the module pgwar entity class as described here: https://kleiolab.atlassian.net/wiki/spaces/ORG/pages/3735781399/Warehouse+entity+class+project+communtiy

### CODE REVIEW

File: database/migrations/sqls/20240627193836-pgwar-entity-class-up.sql [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                    COALESCE(meta.class_label, meta_en.class_label) AS class_label,
+                    COALESCE(meta.entity_type, meta_en.entity_type) AS entity_type,
+                    COALESCE(meta.dfh_parent_classes, meta_en.dfh_parent_classes) AS dfh_parent_classes,
+                    COALESCE(meta.dfh_ancestor_classes, meta_en.dfh_ancestor_classes) AS dfh_ancestor_classes,
+                    COALESCE(meta.class_modified, meta_en.class_modified) AS class_modified,
+                    project_lang.project_modified,
+                    project_lang.language_modified,
+                    ep.fk_class_modified 
+        FROM 		project_lang
+        JOIN		pgwar.entity_preview ep ON project_lang.fk_project = ep.fk_project
```
Comment (dferhod) [inline database/migrations/sqls/20240627193836-pgwar-entity-class-up.sql:73]: Index on entity_preview.fk_project ?

**Review by dferhod — COMMENTED** [review]
See comments for down script and indexes

File: database/migrations/sqls/20240627193836-pgwar-entity-class-up.sql [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    ),
+     project_lang AS (
+    -- select project/community id and its language
+        SELECT 
+            proj.pk_entity as fk_project,
+            COALESCE(TRIM(iso6391), 'en') lang_code, -- language code or english
+            proj.tmsp_last_modification AS project_modified,
+            lang.tmsp_last_modification AS language_modified
+        FROM projects.project proj
+        LEFT JOIN information.language lang ON proj.fk_language = lang.pk_entity
```
[the remark this answers is not in the document]
Comment (joschne) [reply database/migrations/sqls/20240627193836-pgwar-entity-class-up.sql:32]: Yes

File: database/migrations/sqls/20240627193836-pgwar-entity-class-up.sql (hunk 1 above)
Comment (joschne) [reply database/migrations/sqls/20240627193836-pgwar-entity-class-up.sql:73]: Yes

File: database/migrations/sqls/20240627193836-pgwar-entity-class-down.sql
[the remark this answers is not in the document]
Comment (joschne) [reply database/migrations/sqls/20240627193836-pgwar-entity-class-down.sql:1]: done

**Pull request merged**